astropy/modeling/fitting.py

```diff
@@ -1068,3 +1070,35 @@ def _validate_model(model, supported_constraints):
 
     model_copy = model.copy()
     return model_copy
+
+
+def populate_entry_points(entry_points):
```

Comment:
This seems to me that it could be generalized a bit more and moved to `utils` instead?